src/prompt.ts

```diff
@@ -81,6 +81,9 @@ function normal(options: IPromptConfig, retries = 100): Promise<string> {
         resolve(data || options.default)
       }
     })
-    setTimeout(() => reject(), options.timeout || 10000).unref()
+    setTimeout(() => {
+      process.stdin.end()
```

Comment:
missing piece. 🍕

Reply:
this would prevent us from reading stdin ever again in this process which seems problematic

Reply:
would `process.stdin.pause()` work?